include/z64subs.h

```diff
@@ -11,9 +11,9 @@ extern Vec3f gOneVec3f;
 #define SUBS_TIME_PATHING_ORDER 3
 
 typedef enum {
-    /* 0 */ SUBS_CUTSCENE_SET_UNK_LINK_FIELDS,
+    /* 0 */ SUBS_CUTSCENE_PLAYER_CS,
```

Comment:
This one is tricky, because this is dependent on the `playerCsMode` system that's developed by player, but there's been talks of finding a better name for that in general. But I suppose `SUBS_CUTSCENE_PLAYER` is general enough that it should be fine, and is a little better than UNK

Reply:
Perhaps add `With` to match the function name?
i.e `SUBS_CUTSCENE_WITH_PLAYER`?

Reply:
Works for me